VCFv4.3.tex

```diff
@@ -427,9 +427,9 @@ \subsubsection{Genotype fields}
       GT		& 1		& String	& Genotype \\
       HQ		& 2		& Integer	& Haplotype quality \\
       MQ		& 1		& Integer	& RMS mapping quality \\
-      PL		& G		& Integer	& Phred-scaled genotype likelihoods rounded to the closest integer \\
-      PP		& G		& Integer	& Phred-scaled genotype posterior probabilities rounded to the closest integer \\
-      PQ		& 1		& Integer	& Phasing quality \\
+      PL		& G		& Integer	& $-10 log_{10}$ scaled genotype likelihoods rounded to the closest integer \\
```

Comment:
Formatting nits: use `\log` to print a function in roman (and set off by thinspace from the -10) rather than badly spaced single-letter variables in italics; IMHO it looks better keeping the hyphen before `-scaled` (and it's typeset very differently from a minus sign so shouldn't be confusing).

There's already a bunch of `10 log 10` in _VCF*.tex_ that are missing the `\log`, but this PR adds a bunch more instances so should probably use `\log` in its additions.

```suggestion
      PL		& G		& Integer	& $-10\log_{10}$-scaled genotype likelihoods rounded to the closest integer \\
```

Reply:
Bah just noticed the "-scaled" bit too... I'll update again. :-)